### ISSUE TITLE
Document action usage

### ISSUE THREAD

**Issue by Huy-Ngo** [Huy-Ngo/gha-sha#1]
Document how to use this action

**Comment by dinhanhx** [comment]
I suggest using a virtual machine from Google Cloud Platform as a server to host document website. And a virtual machine from Amazon Web Service as an API server querying from database server which is hosted by Azure Microsoft.

**Comment by Huy-Ngo** [comment]
No, thanks. README.md works for this tiny project and is more portable.

**Comment by Huy-Ngo** [comment]
(It's not like it *needs* to be more portable, since it's locked in anyway)

**Comment by dinhanhx** [comment]
I know it doesn't really need a documentation. That's why I suggested a complicated solution.
